spec/dummy/config/environments/test.rb

```diff
@@ -10,6 +10,6 @@
   config.action_controller.allow_forgery_protection = false
   config.action_mailer.default_url_options = { host: 'localhost:3000' }
   config.action_mailer.delivery_method = :test
-  config.action_mailer.perform_deliveries = false
+  config.action_mailer.perform_deliveries = true
```

Comment:
@jayroh actually, there weren't any tests of deliveries per se, only mocking tests (or tests of delivery side-effects). I found these too much work, easier to let it happen and test the deliveries.